al_aws_collector.js

```diff
@@ -175,14 +184,28 @@ class AlAwsCollector {
                         // when all else fails, stringify it the gross way with inspect
                         util.inspect(error);
             }
-            const status = this.prepareErrorStatus(errorString);
+            // post stream specific error
+            const status = streamType ? this.prepareErrorStatus(errorString,'none' , streamType) : this.prepareErrorStatus(errorString);
             this.sendStatus(status, () => {
                 context.fail(errorString);
             });
         } else {
             return context.succeed();
         }
     }
+    prepareHealthyStatus(streamType, streamName = 'none') {
```

Comment:
Inconsistent usage of streamType and collectionType naming between this and prepareErrorStatus functions

Reply:
TO make consistence updated  parameter name to collectionType  in prepareHealthyStatus()